default.json

```diff
@@ -56,8 +72,12 @@
         "mcr.microsoft.com/dotnet/aspnet",
         "mcr.microsoft.com/dotnet/runtime",
         "mcr.microsoft.com/dotnet/runtime-deps",
+        "Azure.Identity",
         "/^[mM]icrosoft\\./",
-        "/^[sS]ystem\\./"
+        "/^[sS]ystem\\./",
+        "coverlet.collector",
+        "xunit",
+        "/^xunit\\./"

```

Comment:
Azure.Identity's API contract is very stable and should stay up-to-date.

Also, updating xunit.* and coverlet.collector is so annoying, it should be automated.

Reply:
Do we want to do the same for a few other packages such as MongoDB.Driver, or maybe security sensitive packages such as BouncyCastle

Reply:
I agree for MongoDB. We don't seem to use BouncyCastle (aside from an older version in .NET Framework).